### DISCUSSION TITLE
Error: The open file operation failed to connect to the open and save panel service.

### DISCUSSION THREAD

**Discussion by sappy96** [RomanKubiak/ctrlr#403] [Q&A]
Hi,

I get this error when trying to save an instance in standalone and VST - "The open file operation failed to connect to the open and save panel service."

Running BigSur Apple M1 and have tried a couple of different versions of CTRLR (5.3.163 and 5.6.23) but all give the same result. I also had a look on google to see if I could find any information but nothing I found seemed specific to my issue.

I'm not a dev so have just been using whatever prebuilds I could find. Any pointers of what the issue might be or if theres an other version I should be using would be appriciated.

Cheers.

**Comment by bijlevel** [comment]
Try it on an earlier MacOS version and let us know how that went. I'm on Mojave and no problems with saving instances.

**Reply by sappy96** [reply]
No worries, if its an OS compatability thing I'll look into spinng up a VM to get around the issue.

**Reply by sappy96** [reply]
It seems you cant run Mojave on an M1 machine, i tried with VM parallels. I got win11 working fine but the exported vst.3 isnt recognised in Ableton or Bitwig when you bring it back to the MACOS. Which probably makes sense as it was created under windows. Looks like i'll need and intel machine to get Mojave running.

**Comment by Godlike-Productions** [comment]
I suspect we'll need a version of ctrlr compiled on an M1, though I can't imagine what build errors may come up while attempting this.

This is good to know though, as I'm looking at an M1 for future panel development.

**Reply by spiffo-smith** [reply]
I'm getting the same problem with v5.3.198 on Monterey 12.6.1

Can't Export any Restricted Instances on my Macbook Air at the moment :-(

Does anyone know which is the latest Mac OS X that doesn't have this problem and / or if there's a later version of Ctrlr that would work? Thanks!

**Reply by sappy96** [reply]
Hey folks,

I just wanted to update that this issue has been resolved for me. Its not a problem with CTRLR, its a MacOS issue.

Something about how they changed the file system folder structure.

It was solved by another user, and if you change preferences in CTRLR to use CTRLRs native file explorer you can export VSTs and Instruments as normal.

Go to CTRLR preferences > Turn of "Use natiove OS file/save option"

I think that was the only setting, but I can't find the thread that mentioned this. Its here somewhere

Hope that helps

**Reply by spiffo-smith** [reply]
Great a simple solution, works fine on Monterey!

I found a couple of quirks in Ableton Live Lite when trying to Export a VST:

1    The ‘Write new instance here’ dialog box appeared directly beneath the Ctrlr Window where I couldn’t see it, drag the Ctrlr Window down to see it!

2    Ableton Live appears not to have access to write to the System VST Folder. So you need to back out to the /Users/ folder somewhere, remove the filename from the suggested filename window (not sure why?) then Press Open
